jax/numpy/lax_numpy.py

```diff
@@ -3787,3 +3788,115 @@ def _unstack(x):
     raise ValueError("Argument to _unstack must be non-scalar")
   return [lax.index_in_dim(x, i, keepdims=False) for i in range(x.shape[0])]
 setattr(DeviceArray, "_unstack", _unstack)
+
+
+# Syntactic sugar for scatter operations.
+class IndexUpdateHelper(object):
```

Comment:
Prefix this name with an underscore. Otherwise I think it will appear in the public namespace of `jax.numpy`.

Also, no need to inherit from `object` now that we're Python 3 only :)